app/tests/scripts/get_user_data.py

```diff
@@ -0,0 +1,48 @@
+import json
+import sys
+from datetime import datetime
+
+import tweepy  # type: ignore
+
+from app.core.config import settings
+
+auth = tweepy.AppAuthHandler(settings.CONSUMER_KEY, settings.CONSUMER_SECRET)
+api = tweepy.API(auth, wait_on_rate_limit=True)
+api_v2 = tweepy.Client(bearer_token=auth._bearer_token, wait_on_rate_limit=True)
+
+if len(sys.argv) != 3:
+    sys.exit("python -m app.tests.script.<current-file> <username> <tweet_num>")
```

Comment:
@produdez can u put an example in `README.md` usage guide :V

Reply:
:v later. Current use: `python -m app.tests.scripts.get_user_data.py deadpool 100`